src/containers/Identity/Home/View.js

```diff
@@ -63,19 +72,26 @@ const Home = (props) => {
           ))}
         </View>
       </ScrollView>
+      <AttestationDetails
+        visible={attestationModalVisibility}
+        attestedClaimName={attestedClaimName}
+        identityAttested={identityAttested}
+      />
     </View>
   );
 };
 
 const mapStateToProps = (state) => ({
   activeIdentity: selectActiveIdentity(state),
   pinnedAttestations: selectPinnedAttestations(state),
+  attestationModalVisibility: selectAttestationModalVisibility(state),
```

Comment:
This sounds like it should be 'visible' or 'hidden', not true or false.
Try naming it showAttestationModal or something like that